src/main.rs

```diff
@@ -36,23 +41,28 @@ fn publish(docs_path: &str,
            sign: bool,
            push_remote: &str,
            push_branch: &str,
+           shell: &mut MultiShell,
            dry_run: bool)
            -> Result<bool, FatalError> {
-    println!("Publishing Sphinx docs to GitHub Pages.");
+    try!(shell.say("Publishing Sphinx docs to GitHub Pages.", color::BLUE));
```

Comment:
Should be an `F: FnMut(&mut MultiShell) -> CargoResult<()>` inside the `verbose`. What about:

```
try!(shell.verbose(|s| s.say(...)));
```